hasura/graphql-engine
# docs: fix relay id (close #6001)

### Description



The relay ID is wrong [here](https://github.com/hasura/graphql-engine/blob/4062cdcbeabd10541f9e51e8b76b97f079f19b00/docs/graphql/core/schema/relay-schema.rst#example) in the docs.

### Changelog


### Affected components



### Related Issues




#6001


### Steps to test and verify



I know the docs is wrong because I ran the following code with my own data and got the right values. Run the following code below to verify:

```sql
SELECT encode('[1, "public", "author", "296d30b1-474d-4011-a907-2701992b04c1"]', 'base64');

-- output
-- WzEsICJwdWJsaWMiLCAiYXV0aG9yIiwgIjI5NmQzMGIxLTQ3NGQtNDAxMS1hOTA3LTI3MDE5OTJiMDRjMSJd

-- docs output
-- WzEsICJwdWJsaWMiLCAiYXV0aG9yIiwgIjI5NmQzMGIxLTQ3NGQtNDAxMS1hOTA3LTI3MDE5OTJiMDRjMSJdCg==
```


#### Catalog upgrade

Does this PR change Hasura Catalog version?


#### Metadata


Does this PR add a new Metadata feature?


#### GraphQL

#### Breaking changes
